extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -921,6 +921,51 @@ tags: {  # <amp-story-page-attachment> with no href
     value: "light"
   }
 }
+tags: {  # Same code as `<amp-story-page-attachment> with href`
+  html_format: AMP
+  tag_name: "AMP-STORY-PAGE-OUTLINK"
+  spec_name: "amp-story-page-outlink"
+  mandatory_ancestor: "AMP-STORY-PAGE"
+  mandatory_last_child: true
+  attrs: {
+    name: "layout"
+    mandatory: true
+    value: "nodisplay"
+  }
+  attrs: {
+    name: "theme"
+    value: "custom"
+    value: "dark"
+    value: "light"
+  }
+  attrs: {
+    name: "cta-accent-color"
+  }
+  attrs: {
+    name: "cta-accent-element"
+    value: "background"
+    value: "text"
+  }
+  attrs: {
+    name: "href"
+    mandatory: true
+    value_url: {
+      protocol: "http"
+      protocol: "https"
+    }
+  }
+  attrs: {
+    name: "cta-image"
```

Comment:
Aren't the images only for the inline attachment?

Reply:
It's for both. 
In this case it's used for a custom preview icon:
<img width="388" alt="Screen Shot 2021-05-03 at 1 01 13 PM" src="https://user-images.githubusercontent.com/3860311/116907554-bc62a380-ac0f-11eb-9f08-507b6d03d666.png">